app/src/main/java/de/westnordost/streetcomplete/util/ktx/LocalDate.kt

```diff
@@ -1,12 +1,31 @@
+@file:Suppress("NOTHING_TO_INLINE")
+
 package de.westnordost.streetcomplete.util.ktx
 
-import java.time.Instant
-import java.time.LocalDate
-import java.time.Month
-import java.time.ZoneId
+import kotlinx.datetime.Clock
+import kotlinx.datetime.DateTimeUnit
+import kotlinx.datetime.atStartOfDayIn
+import kotlinx.datetime.minus
+import kotlinx.datetime.plus
+import kotlinx.datetime.toInstant
+import kotlinx.datetime.toLocalDateTime
+
+typealias Instant = kotlinx.datetime.Instant
+
+typealias LocalDate = kotlinx.datetime.LocalDate
+
+typealias LocalDateTime = kotlinx.datetime.LocalDateTime
+
+typealias LocalTime = kotlinx.datetime.LocalTime
+
+typealias Month = kotlinx.datetime.Month
+
+typealias ZoneId = kotlinx.datetime.TimeZone
+
+typealias ZoneOffset = kotlinx.datetime.FixedOffsetTimeZone
```

Comment:
What are these typealiases for? Why not simply import them?

If some of this is to keep the same names for classes in other files - this is not necessary. Transitioning to kotlinx-datetime is intended to be a one-way-road, we do not intend to go back to java.time. Hence, the API does not need to be the same, just not more complicated (an less complicated is fine too!)

Reply:
I only wanted to keep the diff small. I thought about inlining them in a third commit but decided to wait in case a lot of changes were requested.